minimesos/src/main/java/com/containersol/minimesos/mesos/ClusterArchitecture.java

```diff
@@ -165,7 +165,7 @@ public ClusterContainers getContainers() {
         /**
          * Includes the default {@link ZooKeeperContainer} instance in the cluster
          *
-         * @return TODO(mwl)
+         * @return The modified builder instance (not a copy)
```

Comment:
Stealing from an already existing documentation. StringBuilder.insert says
> a reference to this object.
https://docs.oracle.com/javase/7/docs/api/java/lang/StringBuilder.html#insert(int,%20char[])
IMO it seems a little bit clearer